cyberG/server_ca.crt

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
+MIICXzCCAeSgAwIBAgIUQm8nrECtJnmFQ/vImXOBi/yf3sAwCgYIKoZIzj0EAwMw
+ZjELMAkGA1UEBhMCRkwxETAPBgNVBAcMCEhlbHNpbmtpMRAwDgYDVQQKDAdIZXR6
+bmVyMRAwDgYDVQQDDAdoZXR6TnltMSAwHgYJKoZIhvcNAQkBFhE4OHZnazg4QGdt
+YWlsLmNvbTAeFw0yMjEyMDUxODMxMTNaFw0yNDEyMDQxODMxMTNaMGYxCzAJBgNV
+BAYTAkZMMREwDwYDVQQHDAhIZWxzaW5raTEQMA4GA1UECgwHSGV0em5lcjEQMA4G
+A1UEAwwHaGV0ek55bTEgMB4GCSqGSIb3DQEJARYRODh2Z2s4OEBnbWFpbC5jb20w
+djAQBgcqhkjOPQIBBgUrgQQAIgNiAATd+jqRf+VgiAu12Ajee5O/2w/cNFP8ktve
+lceCf0aQ/YJdjHjRn1vYkkBDn9aqaSCrgYdiKTFheiPNH1Xz9dZY+uLd64zPH2QW
+qIh2ymUhnpNi0paqU6Rne5GClLMCGkajUzBRMB0GA1UdDgQWBBRvSlGWypN0DB7s
+i82/dsVVysGRuTAfBgNVHSMEGDAWgBRvSlGWypN0DB7si82/dsVVysGRuTAPBgNV
+HRMBAf8EBTADAQH/MAoGCCqGSM49BAMDA2kAMGYCMQDiacXkYSYBBJhW1NSI/fmF
+G407DQ3q7/Nd/+SUtalMtoZHnS7TqEzlBK2/hIm+R6ICMQDJYNON3uKC4rsuYyz8
+mb/heaPYbdKqtFGngGOE+qNL8tPiyrdmPvlGWh5Rv6FYFg0=
+-----END CERTIFICATE-----
```

Comment:
Thanks for the PR @Vgk88. From inspection, this appears to be a cert generated by a third party? Please follow the steps outlined here to generate a self-signed CA and use that value. 

https://github.com/PeggyJV/steward/blob/main/docs/04-GeneratingCertificates.md